tests/test_samples.py

```diff
@@ -828,8 +830,9 @@ def test_beta():
 def test_beta_with_logL_infinities():
     ns = read_chains("./tests/example_data/pc")
     ns.loc[:10, ('logL', r'$\ln\mathcal{L}$')] = -np.inf
-
-    ns.recompute(inplace=True)
+    with warnings.catch_warnings(record=True) as w:
+        ns.recompute(inplace=True)
+        assert "samples have logL <= logL_birth" in str(w[0].message)
```

Comment:
For consistency, I'd go with pytest's own handling of catching warnings:
```python
with pytest.warns(UserWarning):
    ns.recompute(inplace=True)
```

Reply:
Done (actually missed this one, github thought it would be helpful to hide three of these conversations)

btw is a `RuntimeWarning`, not `UserWarning`

Reply:
Oh, you're right, since we raise it as a `RuntimeWarning` we should definitely also test for a `RuntimeWarning` and not for a `UserWarning`. 

As to which one would be more appropriate here... never 100% sure about this. I might actually be in favour of having an `AnestheticWarning` for anesthetic specific warnings, but that is definitely beyond this PR.